pheno005/phenotype/ace.json

```diff
@@ -0,0 +1,99 @@
+{
+	"participant_id": {
+		"Description": "OpenNeuro ID of the subject."
+	},
+	"session_id": {
+		"Description": "ID assigned to participant's visit.",
+		"Levels": {
+			"baseline": "First imaging and phenotype data collection visit.",
+			"followup": "Second phenotype data collection visit."
```

Comment:
These should be ses-baseline and ses-followup, respectively. The same should be in the TSV.